<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>561</width>
    <height>406</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QFrame" name="optionView">
        <property name="frameShape">
         <enum>QFrame::Panel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QLabel" name="label">
             <property name="font">
              <font>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Option View</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="hookOptionButton">
             <property name="text">
              <string>Hook</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="hookCollectionOptionButton">
             <property name="text">
              <string>Hook Collection</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="livePacketOptionButton">
             <property name="text">
              <string>Live Packet</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="packetFromPCAPViewOptionButton">
             <property name="text">
              <string>Packet from PCAP View</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="contentView">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QStackedWidget" name="stackedContentView">
           <property name="currentIndex">
            <number>0</number>
           </property>
           <widget class="QWidget" name="page_5">
            <layout class="QHBoxLayout" name="horizontalLayout_4"/>
           </widget>
           <widget class="QWidget" name="page_6"/>
           <widget class="QWidget" name="page_7"/>
           <widget class="QWidget" name="page_8"/>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>561</width>
     <height>18</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExecution_Sequence_Error"/>
    <addaction name="actionProxy_Behavior_Disabled"/>
    <addaction name="actionQueue_Error_Message"/>
    <addaction name="actionHook_Collection_Execution_Sequence_Error"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionExecution_Sequence_Error">
   <property name="text">
    <string>Hook Execution Sequence Error</string>
   </property>
  </action>
  <action name="actionProxy_Behavior_Disabled">
   <property name="text">
    <string>Proxy Behavior Disabled</string>
   </property>
  </action>
  <action name="actionQueue_Error_Message">
   <property name="text">
    <string>Queue Error Message</string>
   </property>
  </action>
  <action name="actionHook_Collection_Execution_Sequence_Error">
   <property name="text">
    <string>Hook Collection Execution Sequence Error</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
